2021-06-17 20:30:07,203:	ERROR:	hcController.py:HcCheckMqttConnect():123	Error when ping to mqtt
2021-06-17 20:30:07,206:	ERROR:	mqttServices.py:MqttConnect():91	Exception in connect to mqtt: [Errno 111] Connection refused
2021-06-17 20:30:12,210:	ERROR:	mqttServices.py:MqttConnect():91	Exception in connect to mqtt: [Errno 111] Connection refused
2021-06-17 20:30:17,216:	ERROR:	mqttServices.py:MqttConnect():91	Exception in connect to mqtt: [Errno 111] Connection refused
2021-06-17 20:30:22,219:	ERROR:	mqttServices.py:MqttConnect():91	Exception in connect to mqtt: [Errno 111] Connection refused
2021-06-17 20:31:23,378:	ERROR:	hcController.py:HcCheckMqttConnect():123	Error when ping to mqtt
2021-06-17 20:31:23,380:	ERROR:	mqttServices.py:MqttConnect():91	Exception in connect to mqtt: [Errno 111] Connection refused
2021-06-17 20:31:54,967:	ERROR:	hcController.py:HcCheckMqttConnect():123	Error when ping to mqtt
2021-06-17 20:31:54,969:	ERROR:	mqttServices.py:MqttConnect():91	Exception in connect to mqtt: [Errno 111] Connection refused
2021-06-17 20:33:53,689:	ERROR:	hcController.py:HcCheckMqttConnect():123	Error when ping to mqtt
2021-06-17 20:33:53,690:	ERROR:	mqttServices.py:MqttConnect():91	Exception in connect to mqtt: [Errno 111] Connection refused
2021-06-17 20:34:22,101:	ERROR:	hcController.py:HcCheckMqttConnect():123	Error when ping to mqtt
2021-06-17 20:34:22,102:	ERROR:	mqttServices.py:MqttConnect():91	Exception in connect to mqtt: [Errno 111] Connection refused
2021-06-17 20:34:22,119:	INFO:	hcController.py:__hcCheckConnectWithCloud():72	Hc send heardbeat to cloud
2021-06-17 20:34:27,106:	ERROR:	mqttServices.py:MqttConnect():91	Exception in connect to mqtt: [Errno 111] Connection refused
2021-06-17 20:36:56,869:	ERROR:	hcController.py:HcCheckMqttConnect():123	Error when ping to mqtt
2021-06-17 20:36:56,870:	ERROR:	mqttServices.py:MqttConnect():91	Exception in connect to mqtt: [Errno 111] Connection refused
2021-06-17 20:36:56,882:	INFO:	hcController.py:__hcCheckConnectWithCloud():72	Hc send heardbeat to cloud
2021-06-17 20:37:01,876:	ERROR:	mqttServices.py:MqttConnect():91	Exception in connect to mqtt: [Errno 111] Connection refused
2021-06-17 20:37:27,093:	ERROR:	hcController.py:HcCheckMqttConnect():123	Error when ping to mqtt
2021-06-17 20:37:27,095:	ERROR:	mqttServices.py:MqttConnect():91	Exception in connect to mqtt: [Errno 111] Connection refused
2021-06-17 20:37:27,116:	INFO:	hcController.py:__hcCheckConnectWithCloud():72	Hc send heardbeat to cloud
2021-06-17 20:37:32,096:	ERROR:	mqttServices.py:MqttConnect():91	Exception in connect to mqtt: [Errno 111] Connection refused
2021-06-17 20:39:40,518:	ERROR:	hcController.py:HcCheckMqttConnect():123	Error when ping to mqtt
2021-06-17 20:39:40,532:	INFO:	hcController.py:__hcCheckConnectWithCloud():72	Hc send heardbeat to cloud
2021-06-17 20:39:45,778:	DEBUG:	mqttServices.py:MqttServicesInit():123	Connect to mqtt status: True
2021-06-17 20:39:51,493:	INFO:	hcController.py:__hcCheckConnectWithCloud():72	Hc send heardbeat to cloud
2021-06-17 20:40:01,066:	DEBUG:	hcController.py:__mqttItemHandler():144	Hc is connecting with mqtt broker
2021-06-17 20:40:02,294:	INFO:	hcController.py:__hcCheckConnectWithCloud():72	Hc send heardbeat to cloud
2021-06-17 20:40:13,865:	INFO:	hcController.py:__hcCheckConnectWithCloud():72	Hc send heardbeat to cloud
2021-06-17 20:40:16,089:	DEBUG:	hcController.py:__mqttItemHandler():144	Hc is connecting with mqtt broker
2021-06-17 20:40:24,570:	INFO:	hcController.py:__hcCheckConnectWithCloud():72	Hc send heardbeat to cloud
2021-06-17 20:40:31,610:	DEBUG:	hcController.py:__mqttItemHandler():144	Hc is connecting with mqtt broker
2021-06-17 20:40:35,057:	INFO:	hcController.py:__hcCheckConnectWithCloud():72	Hc send heardbeat to cloud
2021-06-17 20:40:46,017:	INFO:	hcController.py:__hcCheckConnectWithCloud():72	Hc send heardbeat to cloud
2021-06-17 20:41:06,028:	ERROR:	hcController.py:HcCheckMqttConnect():123	Error when ping to mqtt
2021-06-17 20:41:06,040:	INFO:	hcController.py:__hcCheckConnectWithCloud():72	Hc send heardbeat to cloud
2021-06-17 20:41:11,286:	DEBUG:	mqttServices.py:MqttServicesInit():123	Connect to mqtt status: True
2021-06-17 20:41:26,623:	DEBUG:	hcController.py:__mqttItemHandler():144	Hc is connecting with mqtt broker
2021-06-17 20:41:41,582:	DEBUG:	hcController.py:__mqttItemHandler():144	Hc is connecting with mqtt broker
2021-06-17 20:41:56,638:	DEBUG:	hcController.py:__mqttItemHandler():144	Hc is connecting with mqtt broker
2021-06-17 20:41:57,514:	INFO:	hcController.py:__hcCheckConnectWithCloud():72	Hc send heardbeat to cloud
